pmd-core/src/main/java/net/sourceforge/pmd/cpd/GUI.java

```diff
@@ -631,7 +631,7 @@ private String setLabelFor(Match match) {
             int separatorPos = sourceId.lastIndexOf(File.separatorChar);
             label = "..." + sourceId.substring(separatorPos);
         } else {
-            label = '(' + sourceIDs.size() + " separate files)";
+            label = '(' + String.valueOf(sourceIDs.size()) + " separate files)";
```

Comment:
Wouldn't it be better to use double quotes for the opening parenthesis ?

Reply:
How about `label = String.format("( %d separate files)", sourceIDs.size());`?